deployments/kubernetes/chart/reloader/templates/deployment.yaml

```diff
@@ -76,8 +76,15 @@ spec:
       - image: "{{ .Values.reloader.deployment.image.name }}:{{ .Values.reloader.deployment.image.tag }}"
         imagePullPolicy: {{ .Values.reloader.deployment.image.pullPolicy }}
         name: {{ template "reloader-fullname" . }}
-      {{- if or (.Values.reloader.deployment.env.open) (.Values.reloader.deployment.env.secret) (.Values.reloader.deployment.env.field) (.Values.reloader.deployment.env.existing) (eq .Values.reloader.watchGlobally false) (.Values.reloader.enableHA) (.Values.reloader.enableMetricsByNamespace)}}
         env:
+        - name: GOMEMLIMIT
+          valueFrom:
+            resourceFieldRef:
+              resource: limits.memory
+        - name: GOMAXPROCS
+          valueFrom:
+            resourceFieldRef:
+              resource: limits.cpu
```

Comment:
by default, cpu limits is set to 150m. If you set GOMAXPROCS to limits.cpu, I believe it would round off to 1 (please confirm). 
I see some interesting discussions on this: https://github.com/uber-go/automaxprocs/issues/54

Reply:
K8s will do `ceil` on it - I've tested it using one of the discussion example (or even any empty pod) so 150m will still show as 1. That being said, setting limits fraction of CPU is an antipattern, so I'll adjust the example to set it to 1.

Reply:
Awesome! glad you have tested this.